relstorage/storage.py

```diff
@@ -1073,6 +1073,9 @@ def _abort(self):
         if self.blobhelper is not None:
             self.blobhelper.abort()
 
+    def afterCompletion(self):
+        self._rollback_load_connection()
```

Comment:
I would think so, yeah. But the asymmetry sticks out to me, and it seems like the sort of thing that will get questioned in the future. Perhaps a comment is all that's necessary.